frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1647,12 +1663,11 @@ mod feasibility_check {
 			let raw = raw_solution();
 
 			assert_eq!(raw.solution.unique_targets().len(), 4);
-			assert_eq!(MultiPhase::desired_targets().unwrap(), 8);
+			// desired_targets is capped to the number of targets which is 4
+			assert_eq!(MultiPhase::desired_targets().unwrap(), 4);
 
-			assert_noop!(
-				MultiPhase::feasibility_check(raw, COMPUTE),
-				FeasibilityError::WrongWinnerCount,
-			);
+			// It should succeed
+			assert_ok!(MultiPhase::feasibility_check(raw, COMPUTE));
```

Comment:
this test should be renamed `desired_targets_gets_capped`, and we need a test that actually triggers the call path of `WrongWinnerCount`.

Reply:
That is fixed, can you please let me know if this is what you had in mind to trigger `WrongWinnerCount`?
I have also removed what looks like an outdated comment around the line that triggers the `WrongWinnerCount` error.